packages/nuxt/src/utils/create-ts-config.ts

```diff
@@ -0,0 +1,63 @@
+import { Tree, writeJson } from '@nx/devkit';
+import * as shared from '@nx/js/src/utils/typescript/create-ts-config';
+
+export function createTsConfig(
+  host: Tree,
+  projectRoot: string,
+  type: 'app' | 'lib',
+  options: {
+    strict?: boolean;
+    style?: string;
+    rootProject?: boolean;
+    unitTestRunner?: string;
+  },
+  relativePathToRootTsConfig: string
+) {
+  const json = {
+    compilerOptions: {
```

Comment:
Here, we put all these, or do we just create a `tsconfig.json` file that has:
```
{
  // https://nuxt.com/docs/guide/concepts/typescript
  "extends": "./.nuxt/tsconfig.json"
}

```

Like an original generated `nuxt` app has? I mean, are we missing something? I copied the compilerOptions from there, fwiw

Reply:
made the change